cnd/src/seed.rs

```diff
@@ -85,4 +233,60 @@ mod tests {
         let debug = format!("{:?}", seed);
         assert_eq!(debug, "Seed([*****])".to_string());
     }
+
+    // 32 bytes base64 encoded.
+    const PEM: &str = "-----BEGIN SECRET SEED-----
+syl9wSYaruvgxg9P5Q1qkZaq5YkM6GvXkxe+VYrL/XM=
+-----END SECRET SEED-----
+";
+
+    #[test]
+    fn pem_library_works_as_expected() {
+        let pem = pem::parse(PEM).unwrap();
+        assert_eq!(pem.tag, "SECRET SEED");
+    }
+
+    #[test]
+    fn seed_from_pem_works() {
+        let want = base64::decode("syl9wSYaruvgxg9P5Q1qkZaq5YkM6GvXkxe+VYrL/XM=").unwrap();
+        let pem = pem::parse(PEM).unwrap();
+        let got = Seed::from_pem(pem).unwrap();
+        assert_eq!(got.0, *want);
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
:question: Would it be better to match against the error instead of panicking? Again, I don't know what's best. Maybe this is fine!

Reply:
Nice idea, code is a bit more verbose but more accurate.